Assets/Scripts/Managers/AdultManager.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class AdultManager : SingletonBehaviour<AdultManager>
+{
+	[SerializeField] GameObject _adultPrefab = null;
+
+	[Range( 0.0f, 1.0f )]
+	[SerializeField] float _goodBadCutoff = 0.5f;
+
+	private List<AdultSpawnTag> _spawnPoints = new List<AdultSpawnTag>();
+	private List<ResourceSpawner> _resourceSpawners = new List<ResourceSpawner>();
+
+	void Start()
+	{
+		// Populate list of spawn points.
+		_spawnPoints.AddRange( GameObject.FindObjectsOfType<AdultSpawnTag>() );
+		_resourceSpawners.AddRange( GameObject.FindObjectsOfType<ResourceSpawner>() );
+	}
+
+	public static void SpawnAdult( BuddyStats buddyStats )
+	{
+		instance._SpawnAdult( buddyStats );
+	}
+
+	private void _SpawnAdult( BuddyStats buddyStats )
+	{
+		if ( buddyStats.happiness > _goodBadCutoff )
+		{
+			// Spawn good buddy.
+			AdultSpawnTag spawnTag = _spawnPoints[Random.Range( 0, _spawnPoints.Count )];
+			_spawnPoints.Remove( spawnTag );
+			Transform spawnTransform = spawnTag.GetComponent<Transform>();
+			Instantiate( _adultPrefab, spawnTransform.position, spawnTransform.rotation );
+		}
+		else
+		{
+			// Spawn bad buddy.
+			int index = Random.Range( 0, _resourceSpawners.Count );
+			ResourceSpawner resourceSpawner = _resourceSpawners[index];
+			_resourceSpawners.RemoveAt( index );
+
+			// Deactive spawner once bad buddy is sitting on it.
+			resourceSpawner.enabled = false;
```

Comment:
We don't want to do that because the resources it spawns are children of the resource spawner object, so that would make the resources disappear and we might want to leave a couple of them around. Just disabling the script keeps new ones from spawning.